index.html

```diff
@@ -0,0 +1,127 @@
+                    <h1 class="title">RECIPES TRIED BY IPTY</h1>
+                    <p class="description">I would like to share recipes that I have tried making. They have turned out to be amazing and delicious. I have an amazing journey with food and desserts and the love of learning new recipes. Sharing this with all of you is fruitful.</p>
+                    <div class="date_tag">
+                    <h3 class="date">Every Saturday with a new Recipe</h3>
+                    <a href="#" class="tag">@myRecipes</a>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </header>
+
+    <main class="main">
+        <div class="main-page" id="main">
+        <h2 class="recipes">Recipes</h2>
+        <div class="container">
+            <div class="recipe-block">
+                <img src="images/rice-bowl.png" alt="image" class="img">
+                <h3 class="category">Main</h3>
+                <p class="recipe-text">Sweet and yummy</p>
+            </div>
+            <div class="recipe-block">
+                <img src="images/cake.png" alt="image" class="img">
+                <h3 class="category">Desserts</h3>
+                <p class="recipe-text"> Sweet and yummy</p>
+            </div>
+            <div class="recipe-block">
+                <img src="images/pizza-slice.png" alt="image" class="img">
+                <h3 class="category">Snacks</h3>
+                <p class="recipe-text"> Sweet and yummy</p>
+            </div>
+            <div class="recipe-block">
+                <img src="images/pizza-slice.png" alt="image" class="img">
+                <h3 class="category">Appetizer</h3>
+                <p class="recipe-text"> Sweet and yummy</p>
+            </div>
+            <div class="recipe-block">
+                <img src="images/pizza-slice.png" alt="image" class="img">
+                <h3 class="category">Drinks</h3>
+                <p class="recipe-text"> Sweet and yummy</p>
+            </div>
+        </div>
+        <button class="recipe_btn">More Recipes</button>
+        <a href="#">See More</a>
+        </div>    
+    </main>
+
+    <div class="featured">
+        <div>
+            <h2 class="frecipe">Featured Recipes</h2>
+            <img src="images/line.png" alt="line" class="line">
+            <div class="featurecard">
+            </div>
+        </div>
+        <button class="morebtn"><p>MORE</p><img src="images/pink_arrow.png" alt="arrow" class="arrow"></button>
+    </div>
+    <div class="Mypartners">
+        <h3 class="partners">Partner</h3>
+        <img src="images/line.png" alt="line" class="line1">
+        <div class="brands">
+        <h1 class="brand_names">mozilla</h1>
+        <h1 class="brand_names">Google</h1>
+        <h1 class="brand_names2">NAVER</h1>
+        <h1 class="brand_names1">daumkakao</h1>
+        <h1 class="brand_names1">airbnb</h1>
+        </div>
+    </div>
```

Comment:
- [Optional] Consider wrapping the featured section and my partners section in a `<section>` element. This is best practice for semantic HTML.